extensions/wikia/CuratedContent/CuratedContentController.class.php

```diff
@@ -434,6 +434,49 @@ public function getCuratedContentQuality() {
 		}
 	}
 
+	public function setData( ) {
+		global $wgCityId, $wgEnableCuratedContentUnauthorizedSave;
+		$status = false;
+
+		$this->response->setFormat( WikiaResponse::FORMAT_JSON );
+
+		if ( !empty( $wgEnableCuratedContentUnauthorizedSave ) ) {
+			$helper = new CuratedContentHelper();
+			$data = $this->request->getArray( 'data', [] );
+
+			// strip data
+			foreach( $data as &$section ) {
```

Comment:
We're not only doing `unset`s, we also moving label for section from logical one (that's also from `getData` method to same that exists already in `wgVariable`) - so we deliberately removing stuff added in other function.